README.md

```diff
@@ -1,114 +1,101 @@
+
+> Note: For backward compatibility, we still support `csv2cve` command for producing CVEs from csv but we recommend using new `--input-file` command instead.
+
+You can use `--config` option to provide configuration file for the tool. You can still override options specified in config file with command line arguments. See our sample config files in the 
+[test/config](https://github.com/intel/cve-bin-tool/blob/master/test/config/)
+
 The 0.3.1 release is intended to be the last release to officially support
 python 2.7; please switch to python 3.6+ for future releases and to use the
-development tree.  You can check [our CI
-configuration](https://github.com/intel/cve-bin-tool/blob/master/.github/workflows/pythonapp.yml)
-to see what versions of python we're explicitly testing.
+development tree. You can check [our CI configuration](https://github.com/intel/cve-bin-tool/blob/master/.github/workflows/pythonapp.yml) to see what versions of python we're explicitly testing.
 
 This readme is intended to be a quickstart guide for using the tool.  If you
 require more information, there is also a [user manual](MANUAL.md) available.
 
-How it works
-------------
+## How it works
 
 This scanner looks at the strings found in binary files to see if they
 match certain vulnerable versions of the following libraries and tools:
 
-* binutils
-* bluez
-* bzip2
-* curl
-* expat
-* ffmpeg
-* gnutls
-* gstreamer
-* hostapd
-* icu
-* kerberos
-* libcurl
-* libdb
-* libgcrypt
-* libjpeg
-* libnss
-* libtiff
-* ncurses
-* ngnix
-* node.js
-* openssh
-* openssl
-* png
-* python
-* rsyslog
-* sqlite
-* strongswan
-* syslogng
-* systemd
-* varnish
-* xerces
-* xml2
-* zlib
+|          |          |            | Available checkers |            |            |         |
+| -------- | -------- | ---------- | ------------------ | ---------- | ---------- | ------- |
+| avahi    | bash     | bind       | binutils           | bluez      | busybox    | bzip2   |
+| cups     | curl     | dovecot    | expat              | ffmpeg     | freeradius | gimp    |
+| gnutls   | glibc    | gstreamer  | haproxy            | hostapd    | icecast    | icu     |
+| irssi    | kerberos | libarchive | libdb              | libgcrypt  | libjpeg    | libnss  |
+| libtiff  | libvirt  | lighttpd   | mariadb            | memcached  | ncurses    | nessus  |
+| netpbm   | nginx    | node       | openafs            | openldap   | openssh    | openssl |
+| openswan | openvpn  | png        | polarssl_fedora    | postgresql | python     | qt      |
+| radare2  | rsyslog  | samba      | sqlite             | strongswan | syslogng   | systemd |
+| tcpdump  | varnish  | wireshark  | xerces             | xml2       | zlib       |         |
```

Comment:
The reason I have made table was the same because points are taking to much space and I agree list is only going to increase so a solution would be to redirect user to checkers directory.